modules/microshift-uploading-images-to-mirror.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-uploading-container-images-to-mirror_{context}"]
+= Uploading container images to a mirror registry
+
+To use your container images at an air-gapped site, upload them to the mirror registry using the following procedure.
+
+.Prerequisites
+
+* You are logged into a host with access to `microshift-quay`.
+* The `.pull-secret-mirror.json` file is available locally.
+* The `microshift-containers` directory contents are available locally.
+
+.Procedure
+
+. Install the `skopeo` tool used for copying the container images by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y skopeo
+----
+
+. Set environment variables pointing to the pull secret file, local container image directory, and mirror registry URL for uploading the container images:
++
+[source,terminal]
+----
+IMAGE_PULL_FILE=~/.pull-secret-mirror.json
+IMAGE_LOCAL_DIR=~/microshift-containers
+TARGET_REGISTRY=<microshift-quay:8443> <1>
+----
+<1> Replace <microshift-quay:8443> with the host name and port of your mirror registry server.
+
+. Run the following script snippet to upload the container images to the ${TARGET_REGISTRY} mirror registry:
++
+[source,terminal]
+----
+# Use timestamp and counter as a tag on the target images to avoid
```

Comment:
There is something weird about this line. It is not getting the pound sign in the front when I copy/paste. Can you please have a look ?

Reply:
Is there another way to "explain" what blocks of code are doing?

Reply:
If this is working for copy-paste, maybe we should make it of `terminal` type?
https://github.com/openshift/openshift-docs/pull/64852/files#diff-cef1474a740c9760ab6c927fd9f1f7e9befe350ebbf2b377dabd8fe8a616a7bcR36

Reply:
Yep, working on it. Trying to outsmart asciidoc. But I might need to move the comments below the commands.